server/opts.go

```diff
@@ -2351,6 +2355,12 @@ func parseLeafAuthorization(v interface{}, errors *[]error, warnings *[]error) (
 			auth.user = mv.(string)
 		case "pass", "password":
 			auth.pass = mv.(string)
+		case "nkey":
+			nk := mv.(string)
```

Comment:
Is this guaranteed to be `string` or do we want an `nk, ok :=` to avoid a possible panic?

Reply:
Not guaranteed but like above do not test for failed cast, so I think its ok.